pkg/cert/cert.go

```diff
@@ -36,14 +34,14 @@ const (
 //+kubebuilder:rbac:groups="admissionregistration.k8s.io",resources=validatingwebhookconfigurations,verbs=get;list;watch;update
 
 // CertsManager creates certs for webhooks.
-func CertsManager(mgr ctrl.Manager, namespace string, setupFinish chan struct{}) error {
+func CertsManager(mgr ctrl.Manager, namespace string, configServiceName string, configSecretName string, setupFinish chan struct{}) error {
 	// dnsName is the format of <service name>.<namespace>.svc
-	var dnsName = fmt.Sprintf("%s.%s.svc", serviceName, namespace)
+	var dnsName = fmt.Sprintf("%s.%s.svc", configServiceName, namespace)
 
 	return cert.AddRotator(mgr, &cert.CertRotator{
 		SecretKey: types.NamespacedName{
 			Namespace: namespace,
-			Name:      secretName,
+			Name:      configSecretName,
 		},
 		CertDir:        certDir,
```

Comment:
is this the certDir that is supposed to be set by the ControllerWebhook config as well?